tests/yadism/cf/test_cfh_g1.py

```diff
@@ -0,0 +1,153 @@
+import numpy as np
+import pytest
+from eko.thresholds import ThresholdsAtlas
+
+from yadism import observable_name as on
```

Comment:
```suggestion
from yadism import observable_name as obs
```
In this way it is more descriptive.